scripts/nebula-storaged-listener.service

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+SCRIPT_PATH=$(readlink -f $0)
+SCRIPT_DIR=$(dirname ${SCRIPT_PATH})
+export USAGE_INFO="${0} [-v] [-c /path/to/conf] <start|stop|restart|status|kill>"
+${SCRIPT_DIR}/nebula.service $@ storaged-listener
```

Comment:
this way to handle listener is not necessary, checking whether the daemon is listener or not is the option listener_path, so we don't need the para storaged-listener, just  storaged is enough. and we also don't ned to handle storaged-listener in nebula.sevice.

Reply:
I think the scripted way is simpler.